cvat-core/src/api-implementation.js

```diff
@@ -176,6 +176,7 @@
                 search: isString,
                 status: isEnum.bind(TaskStatus),
                 mode: isEnum.bind(TaskMode),
+                dimension: isString
```

Comment:
In case of dimension has to be only 2D and 3D, so it might be better solution to make an enum for it

Reply:
Made changes accordingly.